src/containers/AccountConnection.jsx

```diff
@@ -220,7 +216,10 @@ class AccountConnection extends Component {
 
   render () {
     const { t, existingAccount, connector, fields } = this.props
-    const { submitting, deleting, error, credentialsError, isSuccessTimedOut } = this.state
+    const { submitting, deleting, error, isSuccessTimedOut } = this.state
+    const hasGlobalError = error && error.message && ![
```

Comment:
I like the idea of passing a `error.code` which would be less error prone and would not force us to pass a new property across components